showcase/src/app/demo-ui/pages/input-mask-directives/demo-input-mask-directives.component.html

```diff
@@ -0,0 +1,26 @@
+<h1 class="mat-display-3" translate>SHOWCASE.DEMO.INPUT_MASK_DIRECTIVES.TITLE</h1>
+<section class="stark-section">
+	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="input-mask-directives/text-mask-directive"
+		[exampleTitle]="'SHOWCASE.DEMO.INPUT_MASK_DIRECTIVES.TEXT_MASK.TITLE' | translate"
+	>
+		<div fxLayout="column" fxLayoutAlign="stretch" fxLayout.gt-sm="row wrap" fxLayoutAlign.gt-sm="space-between end" fxLayoutGap="20px">
```

Comment:
`fxLayoutAlign="stretch"` is not needed here I think. On my PR I set this because of a fix(hack) I had to do to make Emoji's look better. 😄

Reply:
True, is not needed... technically. However if I remove it there is an Error thrown by the `@angular/flex-layout` library: `Cannot read property 'split' of undefined`. This is in fact a bug in the version (7.0.0-beta.22) we use: https://github.com/angular/flex-layout/issues/951.

[It has been solved in version @angular/flex-layout 7.0.0-beta.23](https://github.com/angular/flex-layout/commit/01c4148cdea4942bea88a571d94940f1b0ce56bb) but we cannot upgrade to that for the moment due to the Typescript version we use. So I'll keep this for the moment and add a TODO ;)